### PR TITLE
Fixed 8 typos

### PR DESCRIPTION
**Pull request by andriycraft**

### CODE REVIEW

**Comment by rom1504** [comment]
failing tests

please undo breaking changes

File: src/server.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ class Server extends EventEmitter {
   close () {
     Object.keys(this.clients).forEach(clientId => {
       const client = this.clients[clientId]
-      client.end('ServerShutdown')
+      client.end('Server closed')
```
Comment (rom1504) [inline src/server.js:60]: undo

File: test/serverTest.js [hunk 2]
```diff
@@ -385,7 +385,7 @@ for (const supportedVersion of mc.supportedVersions) {
       let count = 2
       server.on('login', function (client) {
         client.on('end', function (reason) {
-          assert.strictEqual(reason, 'ServerShutdown')
+          assert.strictEqual(reason, 'Server closed')
```
Comment (rom1504) [inline test/serverTest.js:388]: undo

File: src/createServer.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ function createServer (options = {}) {
   const optVersion = version === undefined || version === false ? require('./version').defaultVersion : version
 
   const mcData = require('minecraft-data')(optVersion)
-  if (!mcData) throw new Error(`unsupported protocol version: ${optVersion}`)
+  if (!mcData) throw new Error(`Unsupported protocol version: ${optVersion}`)
```
Comment (rom1504) [inline src/createServer.js:34]: undo

File: src/createClient.js [hunk 4]
```diff
@@ -18,14 +18,14 @@ const versionChecking = require('./client/versionChecking')
...
+  assert.ok(options, 'Options is required')
+  assert.ok(options.username, 'Username is required')
   if (!options.version) { options.version = false }
 
-  // TODO: avoid setting default version if autoVersion is enabled
+  // TODO: Avoid setting default version if autoVersion is enabled
   const optVersion = options.version || require('./version').defaultVersion
   const mcData = require('minecraft-data')(optVersion)
-  if (!mcData) throw new Error(`unsupported protocol version: ${optVersion}`)
+  if (!mcData) throw new Error(`Unsupported protocol version: ${optVersion}`)
```
Comment (rom1504) [inline src/createClient.js:28]: undo

Comment (andriycraft) [reply src/createClient.js:28]: why?

File: src/createServer.js (hunk 3 above)
Comment (andriycraft) [reply src/createServer.js:34]: why?

File: src/createClient.js (hunk 4 above)
Comment (rom1504) [reply src/createClient.js:28]: because this is a breaking change with little benefit

File: src/createServer.js (hunk 3 above)
Comment (rom1504) [reply src/createServer.js:34]: because this is a breaking change with little benefit

**Pull request closed without merging**